dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java

```diff
@@ -241,48 +240,56 @@ public String decompressValue(String value, ConnectionContext ctx) {
      * <ul>
      * <lh>String Operations</lh>
      * <li>{@link #mget(String...) MGET}</li>
+     * <li>{@link #mset(String...) MSET}</li>
+     * <li>{@link #msetnx(String...) MSETNX}</li>
      * </ul>
      *
      * @param <T>
      *            the parameterized type
      */
     private abstract class CompressionValueMultiKeyOperation<T> extends MultiKeyOperation<T>
```

Comment:
may be rename this to alternate value Multi key operation etc? just an idea

Reply:
It is not an alternate value. It is a key-value pair.